### PR TITLE
Add iCal parsing to GTT (for EXA related maintenance)

### PR DESCRIPTION
**Pull request by aliex-13**
I noticed in our organisation that EXA was sending BCOP standardised messages but was getting grouped into the GTT class and therefore was not being parsed correctly as all information was in the ICS file... not the HTML.

GTT is still a separate entity which will send their own maintenance notifications and therefore I have made the decision to update this and move EXA away from GTT (even though it is a sub-entity).

Testing scenarios have been added and debugging has been completed.

Updated the GTT class with a default organiser to be an email address which we have had sent maintenance notifications from GTT in the past.

### CODE REVIEW

**Comment by aliex-13** [comment]
Just noticed the README specifies supported vendors etc, it might be worth updating this too to have EXA in the list as a supported BCOP provider?

**Comment by chadell** [comment]
Hi @aliex-13 , as commented offline, we would prefer to leverage the multiple processor composability to keep both providers as one.

**Comment by aliex-13** [comment]
Hi @chadell , I've fixed the mentioned code.

Please take a look when you get a chance :)

File: circuit_maintenance_parser/provider.py [hunk 1]
```diff
@@ -297,10 +297,10 @@ class GTT(GenericProvider):
     """EXA (formerly GTT) provider custom class."""
 
     # "Planned Work Notification", "Emergency Work Notification"
-    _include_filter = PrivateAttr({EMAIL_HEADER_SUBJECT: ["Work Notification"]})
```
[the remark this answers is not in the document]
Comment (aliex-13) [reply circuit_maintenance_parser/provider.py:300]: Hi @chadell , as discussed in our private chat - I can't get this section to work with both iCal formatted data (due to not having an email subject header I believe) and therefore, removing it is the easier option. I am open to figuring out how we can have both data types (GTT HTML processor and iCal processor) with the same include filter above, but just need some guidance. Any help would be appreciated!

[the remark this answers is not in the document]
Comment (aliex-13) [reply circuit_maintenance_parser/provider.py:300]: I believe this thread is resolved with my latest push. All unit tests pass and my testing passes.
Small summary:
Basically, if there is an ical / Calendar object to check in the data, as long as it has a mandatory field of "BEGIN", it should pass the _include_filter. Also for normal emails from GTT that have the existing "Work Notification" field in the subject, this should continue to pass (all existing unit tests pass, so this should work).

[the remark this answers is not in the document]
Comment (chadell) [reply circuit_maintenance_parser/provider.py:300]: many thanks for your contribution!

**Review by chadell — APPROVED** [review]
LGTM

**Pull request merged**